polls.php

```diff
@@ -1150,7 +1158,7 @@
 	$votesarray = explode("||~|~||", $poll['votes']);
 	if(count($votesarray) > $poll['numoptions'])
 	{
-		$votesarray = array_slice(0, $poll['numoptions']);
+		$votesarray = array_slice($poll['numoptions'], 0, $poll['numoptions']);
```

Comment:
Shouldn't the first parameter to `array_slice()` here be `$votesarray` rather than `$poll['numoptions']`?